### DISCUSSION TITLE
Azam Sports HD

### DISCUSSION THREAD

**Discussion by ahmedzooo** [iptv-org/community#676] [Q&A]
**Channel Name:**  azam sports 1
**Channel ID:** AzamSports1.tz
**Country:** Tanzania
**Language:** sawhili
**External Link (optional):** xxx
**Notes (optional):** 
Links on this site no work

**Comment by ahmedzooo** [comment]
Not working this links

**Comment by freearhey** [comment]
@ahmedzooo The broken links have already been removed from the playlist via https://github.com/iptv-org/iptv/commit/41aea0a7cd7126b2a12ea895134956053b5a4290, and the channels themselves have been added to the list (https://github.com/iptv-org/iptv/issues/2730).

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19675
